src/crypto/CHIPCryptoPAL.h

```diff
@@ -71,7 +71,7 @@ const unsigned char spake2p_N_p256[65] = {
 /**
  * Spake2+ state machine to ensure proper execution of the protocol.
  */
-enum
+enum class CHIP_SPAKE2P_STATE
```

Comment:
Do we want to throw in `: uint8_t` at the end to ensure that it stays one-byte as before and the compiler doesn't randomly widen it?